src/audio/pipeline/pipeline-stream.c

```diff
@@ -119,14 +111,22 @@ static int pipeline_comp_trigger(struct comp_dev *current,
 
 	/* send command to the component and update pipeline state */
 	err = comp_trigger(current, ppl_data->cmd);
+	current->pipeline->trigger.aborted = err == PPL_STATUS_PATH_STOP;
```

Comment:
this looks wrong.

Reply:
@lgirdwood sorry, could you explain? I don't see what's wrong with this - if one of components returns `PATH_STOP` we abort trigger processing?